Scripts/RemediationScripts/Remediate-EnableHttpsForStorageAccount.ps1

```diff
@@ -0,0 +1,723 @@
+                    Write-Host "Backing up config of Storage Account(s) details. Please do not delete this file. Without this file you won't be able to rollback any changes done through remediation script." 
+                    $stgWithDisableHTTPs | Export-CSV -Path "$($folderpath)\StorageWithdisableHTTPs.csv" -NoTypeInformation
+                    Write-Host "Path: $($folderPath)\StorageWithdisableHTTPs.csv" -ForegroundColor $([Constants]::MessageType.Update)
+                }
+
+                write-host $([Constants]::SingleDashLine)
+               
+                if (-not $Force)
+                {
+                    Write-Host "Do you want to enable HTTPS on the storage account(s)? " -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
+                    $userInput = Read-Host -Prompt "(Y|N)"
+    
+                    if($userInput -ne "Y")
+                    {
+                        Write-Host "HTTPS will not be enabled for any of the storage account. Exiting..." -ForegroundColor $([Constants]::MessageType.Info)
+                        break
+                    }
+                }
+                else
+                {
+                    Write-Host "'Force' flag is provided. HTTPS will be enabled on the storage account without any further prompts." -ForegroundColor $([Constants]::MessageType.Warning)
+                }
+
+              
+                
+                write-host $([Constants]::DoubleDashLine)
+                Write-Host "[step 6 of 6] : Setting the value of 'EnableHttpsTrafficOnly' to 'true' for the Storage Account(s) of subscription."     
+                $remediationSuccess = @() 
+                    
+                Write-Host "Enabling 'EnableHttpsTrafficOnly' on [$($totalstgWithdisableHTTPs)] Storage Account(s)..."
+                $stgWithDisableHTTPs = $stgWithDisableHTTPs | Sort-Object -Property "ResourceGroupName"
+                $stgWithDisableHTTPs | ForEach-Object {
+                    try
+                    {   
+                        $output = Set-AzStorageAccount -ResourceGroupName $_.ResourceGroupName -Name $_.StorageAccountName -EnableHttpsTrafficOnly $true -ErrorAction SilentlyContinue
+                        if($output -ne $null)
+                        {
+                            $remediationSuccess += $_ 
+                        }   
+                        else
+                        {   
+                            $skippedStorageAccountFromRemediation += $_
+                        }
+                    }        
+                    catch
+                    {
+                       $skippedStorageAccountFromRemediation += $_
+                    }
+                }
+                    
+                    
+                write-host $([Constants]::DoubleDashLine)
+                if(($skippedStorageAccountFromRemediation | Measure-Object).Count -eq 0)
```

Comment:
Can lines 468-487 be rewritten as below to keep things simple?

```
if success count is > 0
    print storage accounts that were successfully remediated/rolled back
if failure count is > 0
    print storage accounts that were not successfully remediated/rolled back
if skipped count is > 0
    print storage accounts that were skipped during remediation/roll back
```

Anything additional that you do to tell an all success/failure is a bonus.